examples/pipeline_example.py

```diff
@@ -1,17 +1,36 @@
 import pandas as pd
 from sklearn.model_selection import train_test_split
-from trees.StrongTree import StrongTreeClassifier
+from trees.FairTree import FairTreeClassifier
```

Comment:
I think it would be good to have examples for each of the tree types. Let's keep this original pipeline example for StrongTree and create a new example for the FairTree classifier. We could call this file `fit_FairTree_example` for example.